apps/example-orchestrator-ui/components/Subscriptions/Subscriptions.tsx

```diff
@@ -0,0 +1,266 @@
+        'insync',
+        'organisationName',
+        'organisationAbbreviation',
+        'productName',
+        'tag',
+        'startDate',
+        'endDate',
+        'notes',
+    ];
+
+    const leadingControlColumns: ControlColumn<Subscription>[] = [
+        {
+            id: 'inlineSubscriptionDetails',
+            width: 40,
+            rowCellRender: () => (
+                <EuiFlexItem>
+                    <PlusCircleFill color={theme.colors.mediumShade} />
+                </EuiFlexItem>
+            ),
+        },
+    ];
+
+    return (
+        <Table
+            data={mapApiResponseToSubscriptionTableData(data)}
+            pagination={{
+                pageSize: pageSize,
+                pageIndex: Math.floor(pageIndex / pageSize),
+                pageSizeOptions: [5, 10, 15, 20, 25, 100], // todo move to constants file
+                totalRecords: 300, // todo get from graphql result
+                onChangePage: (updatedPageNumber) =>
+                    setPageIndex(updatedPageNumber * pageSize),
+                onChangeItemsPerPage: (itemsPerPage) =>
+                    setPageSize(itemsPerPage),
+            }}
+            columns={tableColumnConfig}
+            leadingControlColumns={leadingControlColumns}
+            initialColumnOrder={initialColumnOrder}
+            dataSorting={{
+                columnId: sortedColumnId,
+                sortDirection: mapPythiaSortOrderToSortDirection(
+                    sortOrder.order,
+                ),
+            }}
+            updateDataSorting={(dataSorting) =>
+                setSortOrder({
+                    field: dataSorting.columnId,
+                    order:
+                        dataSorting.sortDirection === SortDirection.Asc
+                            ? PythiaSortOrder.Asc
+                            : PythiaSortOrder.Desc,
+                })
+            }
+        ></Table>
+    );
+};
+
+function mapApiResponseToSubscriptionTableData(
+    graphqlResponse: SubscriptionGridQuery,
+): Subscription[] {
+    return graphqlResponse.subscriptions.edges.map(
+        (baseSubscription: MyBaseSubscriptionEdge): Subscription => {
+            const {
+                description,
+                insync,
+                organisation,
+                product,
+                startDate,
+                endDate,
+                status,
+                subscriptionId,
+                note,
+            } = baseSubscription.node;
+
+            return {
+                description,
+                insync,
+                organisationName: organisation.name,
+                organisationAbbreviation: organisation.abbreviation,
+                productName: product.name,
+                tag: product.tag,
+                startDate,
+                endDate,
+                status,
+                subscriptionId,
+                notes: note,
+            };
+        },
+    );
+}
+
+function mapPythiaSortOrderToSortDirection(
```

Comment:
I would suggest not adding "Pythia" that's an implementation detail for SURF (for now, until the orchestrator graphql stuff is ready).

Reply:
Good one, done